x-pack/legacy/plugins/snapshot_restore/public/app/components/policy_form/steps/step_settings.tsx

```diff
@@ -0,0 +1,450 @@
+                      options.forEach(({ label, checked }) => {
+                        if (checked === 'on') {
+                          newSelectedIndices.push(label);
+                        }
+                      });
+                      setIndicesOptions(options);
+                      updatePolicyConfig({ indices: [...newSelectedIndices] });
+                      setCachedIndices([...newSelectedIndices]);
+                    }}
+                    searchable
+                    height={300}
+                  >
+                    {(list, search) => (
+                      <EuiPanel paddingSize="s" hasShadow={false}>
+                        {search}
+                        {list}
+                      </EuiPanel>
+                    )}
+                  </EuiSelectable>
+                ) : (
+                  <EuiComboBox
+                    options={indices.map(index => ({ label: index }))}
+                    placeholder={i18n.translate(
+                      'xpack.snapshotRestore.policyForm.stepSettings.indicesPatternPlaceholder',
+                      {
+                        defaultMessage: 'Enter index patterns, i.e. logstash-*',
+                      }
+                    )}
+                    selectedOptions={indexPatterns.map(pattern => ({ label: pattern }))}
+                    onCreateOption={(pattern: string) => {
+                      if (!pattern.trim().length) {
+                        return;
+                      }
+                      const newPatterns = [...indexPatterns, pattern];
+                      setIndexPatterns(newPatterns);
+                      updatePolicyConfig({
+                        indices: newPatterns.join(','),
+                      });
+                    }}
+                    onChange={(patterns: Array<{ label: string }>) => {
+                      const newPatterns = patterns.map(({ label }) => label);
+                      setIndexPatterns(newPatterns);
+                      updatePolicyConfig({
+                        indices: newPatterns.join(','),
+                      });
+                    }}
+                  />
+                )}
+              </EuiFormRow>
+            </Fragment>
+          )}
+        </Fragment>
+      </EuiFormRow>
+    </EuiDescribedFormGroup>
+  );
+
+  const renderIgnoreUnavailableField = () => (
+    <EuiDescribedFormGroup
+      title={
+        <EuiTitle size="s">
+          <h3>
+            <FormattedMessage
+              id="xpack.snapshotRestore.policyForm.stepSettings.ignoreUnavailableDescriptionTitle"
+              defaultMessage="Ignore unavailable indices"
+            />
+          </h3>
+        </EuiTitle>
+      }
+      description={
+        <FormattedMessage
+          id="xpack.snapshotRestore.policyForm.stepSettings.ignoreUnavailableDescription"
+          defaultMessage="Ignores indices that are unavailable when taking the snapshot. Otherwise, the entire snapshot will fail."
```

Comment:
@gchaps "Otherwise, the entire snapshot will fail" sounds like "If you don't choose this option the snapshot will fail." Do you think being more explicit by adding "Otherwise, the entire snapshot will fail if any indices are unavailable" is worth it or am I overthinking things?